lightning/src/ln/onion_utils.rs

```diff
@@ -848,49 +843,94 @@ mod tests {
 		assert_eq!(onion_keys[4].rho, hex::decode("034e18b8cc718e8af6339106e706c52d8df89e2b1f7e9142d996acf88df8799b").unwrap()[..]);
 		assert_eq!(onion_keys[4].mu, hex::decode("8e45e5c61c2b24cb6382444db6698727afb063adecd72aada233d4bf273d975a").unwrap()[..]);
 
-		// Test vectors below are flat-out wrong: they claim to set outgoing_cltv_value to non-0 :/
+		// Packet creation test vectors from BOLT 4 (see https://github.com/lightning/bolts/blob/16973e2b857e853308cafd59e42fa830d75b1642/bolt04/onion-test.json).
```

Comment:
Ok @tnull, would you prefer the line like this instead? I'll change it if you think this is better :)

```suggestion
		// Packet creation test vectors from BOLT 4 (see 
		// https://github.com/lightning/bolts/blob/16973e2b857e853308cafd59e42fa830d75b1642/bolt04/onion-test.json).
```

Reply:
This looks a bit off in the github GUI, but basically a new line right before the link.

Reply:
Yeah, I'd prefer moving it to the next line, but don't care too much to be honest.

Reply:
Ok fixed :)